pug/product-page/sign-up/index.pug

```diff
@@ -0,0 +1,53 @@
+section.section-sign-up
+    .container
+        .row
+            .col-lg-12.sign-up-page__heading
+                .category__heading
+                    h3 Trang chủ
+                    i.sign-up-filter__page-icon.fas.fa-caret-right
+                    h3.sign-up-filter__page-heading Đăng kí
+            .col-lg-12
+                .sign-up__container
+                    form.sign-up__form
+                        h1.sign-up__form-title Thông tin cá nhân:
+                        .row
+                            .col-lg-6
+                                .sign-up__form-label
+                                    label Họ và tên:
+                                    span *
+                                input.sign-up__form-input(type="text")
```

Comment:
Bổ sung thuộc tính for của label và id cho input
Tìm hiểu thông qua bài viết này
https://css-tricks.com/html-inputs-and-labels-a-love-story/